docs/architecture/modernize-desktop/whats-new-with-netcore3- for-desktop.md

```diff
@@ -0,0 +1,147 @@
+# What&#39;s new with .NET Core 3.0 for Desktop?
+
```

Comment:
file name here is problematic too. I'd avoid long names and specific versions listed.

Suggestion: whats-new-dotnet-core-desktop.md